docs/engineering/android-app/configuring/config-types/widget.mdx

```diff
@@ -223,6 +223,38 @@ serviceButton | A ButtonProperties object that represents a button displayed in
 services | A list of ButtonProperties that represent the services associated with the service card | yes | null |
 actions | A list of ActionConfig objects that represent the actions that can be performed on the service card | yes | emptyList() |
 
+
+**serviceMemberIcons**<br/><br/>
+One of the config properties of the SERVICE_CARD viewType are the serviceMemberIcons. Some icons provided within fhircore
+for use include the ![BABY_BOY](/img/ic_baby_boy.png), ![BABY_GIRL](/img/ic_baby_girl.png), ![WOMAN_OF_REPRODUCTIVE_AGE](/img/ic_woman_of_reproductive_age.png),
+![POST_PARTUM_MOTHER](/img/ic_post_partum_mother.png), ![ELDERLY](/img/ic_elderly.png), and ![SICK_CHILD](/img/ic_sick_child.png).
+
+Since you could have many icons intended for use, ![it could be better to extract the icons and add a reference to them
```

Comment:
Not sure what the ![ does in this line, assuming it might be a typo?

Reply:
No. without the `!`, it becomes a link. Same way we add links on markdown [description](url)

Reply:
you're right, on line 232 is definitely a typo